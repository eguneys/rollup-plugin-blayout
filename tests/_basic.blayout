App
  Scene
    Dropdown `${drops}` `${selected_index}`
