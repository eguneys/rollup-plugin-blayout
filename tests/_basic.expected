import { View, _ctor } from './views'
import {
  App as App_$,
  Scene as Scene_$,
  Dropdown as _Dropdown_$
} from './views'

let _App = (drops, selected_index) => _ctor(App_$, drops, selected_index)
let _Scene = () => _ctor(Scene_$)
let _Dropdown = (items) => _ctor(Dropdown_$, items)
 

export const App = (drops, selected_index) => {
  let parent = _App(drops, selected_index)
  
  let children = [
    _Scene()
    _Dropdown(drops)
  ]

  children.forEach(_ => _.parent = parent)
  parent.children = children

  return parent
}
